freelist_hmap.go

```diff
@@ -139,6 +135,7 @@ func (f *freelist) delSpan(start pgid, size uint64) {
 	if len(f.freemaps[size]) == 0 {
 		delete(f.freemaps, size)
 	}
+	f.freePagesCount -= size
 }
 
 // initial from pgids using when use hashmap version
```

Comment:
For "completeness", please reset the counter in the init() method.

Reply:
thx for quickly reply~  fix it